video/mp_image.c

```diff
@@ -549,6 +549,12 @@ void mp_image_copy_attributes(struct mp_image *dst, struct mp_image *src)
     assign_bufref(&dst->dovi_buf, src->dovi_buf);
     assign_bufref(&dst->film_grain, src->film_grain);
     assign_bufref(&dst->a53_cc, src->a53_cc);
+
+    dst->ff_side_data = talloc_memdup(NULL, src->ff_side_data,
```

Comment:
```suggestion
    if (dst->ff_side_data) {
        for (int n = 0; n < dst->num_ff_side_data; n++)
            av_buffer_unref(&dst->ff_side_data[n]);
        talloc_free(dst->ff_side_data);
    }
    dst->ff_side_data = talloc_memdup(NULL, src->ff_side_data,
```

Reply:
Indeed, I was thinking about freeing old ones, but then forgot to actually do it.